v8-sys/build.rs

```diff
@@ -97,9 +97,15 @@ fn link_v8() {
     }
 
     if cfg!(feature = "shared") {
-        println!("cargo:rustc-link-lib=dylib=v8");
-        println!("cargo:rustc-link-lib=dylib=icui18n");
-        println!("cargo:rustc-link-lib=dylib=icuuc");
+        if cfg!(all(windows, target_env="msvc")) {
+            println!("cargo:rustc-link-lib=dylib=v8.dll");
+            println!("cargo:rustc-link-lib=static=v8_base");
+            println!("cargo:rustc-link-lib=static=v8_platform");
```

Comment:
So `v8_platform` is needed if you don't implement your own, which we do through the `PlatformFunctions` API towards Rust.